### PR TITLE
Fix path to libSystem.Native.a on Linux

### PR DESCRIPTION
**Pull request by schellap**
@gkhanna79, could you PTAL?

### CODE REVIEW

File: src/Microsoft.DotNet.Tools.Compiler.Native/IntermediateCompilation/Linux/LinuxRyuJitCompileStep.cs [hunk 1]
```diff
@@ -78,10 +78,11 @@ private void InitializeArgs(NativeCompileSettings config)
             }
 
             // AppDep Libs
+            var baseAppDepLibPath = Path.Combine(config.AppDepSDKPath, "CPPSdk/ubuntu.14.04", config.Architecture);
```
Comment (gkhanna79) [inline src/Microsoft.DotNet.Tools.Compiler.Native/IntermediateCompilation/Linux/LinuxRyuJitCompileStep.cs]: Shouldn't config.Architecture be converted to a string for combining? 

Thanks for making the fix.

Comment (schellap) [inline src/Microsoft.DotNet.Tools.Compiler.Native/IntermediateCompilation/Linux/LinuxRyuJitCompileStep.cs]: Ha ha! Right. Thanks btw for the mac support.

**Comment by schellap** [comment]
@dotnet-bot test Windows_NT Release pls

**Comment by schellap** [comment]
@dotnet-bot test this please

**Comment by schellap** [comment]
@brthor merging

**Pull request merged**